Saturation/Splitter.cpp

```diff
@@ -675,9 +675,17 @@ void Splitter::init(SaturationAlgorithm* sa)
 #endif
 
   if (opts.splittingAvatimer() < 1.0) {
-    _stopSplittingAtTime = opts.splittingAvatimer() * opts.timeLimitInDeciseconds() * 100;
-#if VAMPIRE_PERF_EXISTS
-    _stopSplittingAtInst = opts.splittingAvatimer() * opts.instructionLimit();
+    unsigned timeLimit = opts.simulatedTimeLimit(); // is also stored in deciseconds
+    if (timeLimit == 0) {
+      timeLimit = opts.timeLimitInDeciseconds();
+    }
+    _stopSplittingAtTime = opts.splittingAvatimer() * timeLimit * 100;
+#ifdef VAMPIRE_PERF_EXISTS
```

Comment:
`#if VAMPIRE_PERF_EXISTS`, it's always defined.

Reply:
Thanks for noticing this!